<?xml version="1.0"?>

<launch>
  <arg name="rvizconfig" value="$(find eyes_drawer)/config/demo_head.rviz" />

  <!-- Head model publisher -->
  <include file="$(find demo_head)/launch/publish_head.launch" />
  
  <!-- World basepoint -->
  <node pkg="tf" type="static_transform_publisher" name="world_broadcaster" args="0 0 0 0 0 0 world base 100" />
  
  <!-- Point of interest -->
  <node pkg="poi_test_publisher" type="poi_test_publisher.py" name="poi_test_publisher" />

  <!-- Actual eye drawer -->
  <node name="eyes_drawer" pkg="eyes_drawer" type="eyes_drawer_node" required="true">
    <param name="topics_ns" value="eyes" />
    <param name="test_image" value="$(find eyes_drawer)/images/eye_demo.png" />
    <param name="img_width" value="320" />
    <param name="img_height" value="240" />
  </node>


  <!-- Rviz -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
